[16364] Make switching between create views possible

furinvader opened the pull request:
As an addition to the full create view, it is now possible to switch between list an full create while creating a work package.

https://community.openproject.org/work_packages/16364/activity

furinvader committed:
Remove unused onCancel

furinvader committed:
Make switching between list an full create possible

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://furinvader-features-16364-switch-create-views.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://furinvader-features-16364-switch-create-views.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

furinvader commented:
I've encountered an error when using the cancel button.
You get redirected between the create views after you've switched at least once.

furinvader committed:
Fix cancellation error

When cancel creation of a work package after
switching from list to full create, the state was
was switched instead of redirecting the user
to a non creation state.

oliverguenther commented:
Should it be possible to move from fullscreen create to split screen create? I didn't find any related acceptance criteria. It does switch with all entered values by hitting the back button, though.

The other way around works nicely.
